ui/site/css/forum/_reactions.scss

```diff
@@ -50,8 +51,8 @@
   }
 
   button.yes {
+    display: block;
```

Comment:
This `display: block` isn't necessary. `.yes` buttons seem to have `display: flex` already from [_modal.scss](https://github.com/ornicar/lila/blob/master/ui/common/css/component/_modal.scss#L3)

Reply:
Removed it, and changed display: block to flex.